### DISCUSSION TITLE
Tool to find shared selected features

### DISCUSSION THREAD

**Discussion by edugm94** [blue-yonder/tsfresh#919] [Q&A]
Hi everybody,

I am extracting and selecting features from different sets. I have stored the settings file returned from function: `tsfresh.feature_extraction.settings.from_columns()`. I would like to get shared features among which have been computed in the different setting files. 
Is there any built-in method implemented in the library which can tell me that information?

Thanks in advance.

**Comment by MaxBenChrist** [comment]
I don't think so, but I could see the need for a union / diff functionality on those settings objects.

**Reply by edugm94** [reply]
Thanks for the response! Yes it can be an interesting features :)
